jovian/utils/slug.py

```diff
@@ -0,0 +1,14 @@
+from jovian.utils.rcfile import get_notebook_slug
+
+_current_slug = None
+
+
+def set_current_slug(filename):
```

Comment:
This method name is a bit misleading. One would expect that `set_current_slug` will set the `_current_slug` to the given argument.

Might be better to call this `detect_current_slug`